### PR TITLE
Fix layout when press key up/down on windows

### PR DESCRIPTION
**Pull request by tavriaforever**
@gela-d 
I hide slide block and show it only when panel must open.
For saving animate, I changed css from `transition` to `animate` and added `0.1s` delay for set `display: block` before starting the animation. 
p.s. Only for windows os

### CODE REVIEW

File: src/blocks/common.blocks/search-panel/search-panel.styl [hunk 1]
```diff
@@ -12,13 +13,19 @@
...
 
-    transition: right .3s ease-out;
-
     background-color: #fafafa;
     box-shadow: 0 0 2px 0 rgba(0,0,0,.3);
 
     &_state_open
     {
-        right: 0;
+        display: block;
```
Comment (gela-d) [inline src/blocks/common.blocks/search-panel/search-panel.styl:21]: needs line-break

**Comment by gela-d** [comment]
bug: if we try to write smth to search-panel - it closes

**Comment by tavriaforever** [comment]
update

**Comment by tavriaforever** [comment]
@gela-d  ping

**Comment by gela-d** [comment]
все ок

**Pull request merged**